glustercli/cmd/georep.go

```diff
@@ -54,12 +56,24 @@ var (
 	flagGeorepCmdForce        bool
 	flagGeorepShowAllConfig   bool
 	flagGeorepRemoteEndpoints string
+	flagRemoteUser            string
+	flagRemoteSecret          string
+	flagRemoteSecretFile      string
+	flagRemoteCacert          string
+	flagRemoteInsecure        bool
 )
 
 func init() {
 	// Geo-rep Create
 	georepCreateCmd.Flags().StringVar(&flagGeorepRemoteEndpoints, "remote-endpoints", "", "remote glusterd2 endpoints")
 	georepCreateCmd.Flags().BoolVarP(&flagGeorepCmdForce, "force", "f", false, "Force")
+	georepCreateCmd.Flags().StringVar(&flagRemoteUser, "remote-user", "glustercli", "Username for authentication")
+	georepCreateCmd.Flags().StringVar(&flagRemoteSecret, "remote-secret", "", "Password for authentication")
+	georepCreateCmd.Flags().StringVar(&flagRemoteSecretFile, "remote-secret-file", "", "Path to file which contains the secret for authentication")
+	georepCreateCmd.Flags().StringVar(&flagRemoteCacert, "remote-cacert", "", "Path to CA certificate")
+	georepCreateCmd.Flags().BoolVar(&flagRemoteInsecure, "remote-insecure", false,
+		"Accepts any certificate presented by the server and any host name in that certificate.")
```

Comment:
can this be reworded as `Skip remote server certificate validation` something like that?

Reply:
done